packages/grid/test/row-details.test.js

```diff
@@ -294,7 +294,7 @@ describe('row details', () => {
     beforeEach(async () => {
       dataset = buildDataSet(10);
       grid = fixtureSync(`
-        <vaadin-grid style="width: 50px; height: 400px" size="100">
+        <vaadin-grid style="width: 50px; height: 400px" size="10">
```

Comment:
**note:** The grid size didn't match the number of items the data provider actually returned (10).